### ISSUE TITLE
Set up a forum, add a comment system on the website, etc

### ISSUE THREAD

**Issue by moul** [berty/community#85]
_TLDR:_
* We will set up a Discourse Forum,
* add a comment system on our website,
* continue to use Discord as our primary public chat.

---

This morning, we decided multiple things about the tooling of the community

1. We will bootstrap a community forum on Discourse; this forum will be about keeping topic-based conversations that can be discovered more easily than a community chat;
2. We will use its built-it "Embed Comments" system to allow people to let comments on our website, at least on our blog posts and newsletters;
3. Discourse stay our main community chat and for now, we will disconnect the matrix bridge for now and maybe re-enable it later, when channels, roles, and rules will be more stable on our main chat;
4. We will share more things on Discord, including periodic notes about what we are currently working on;
5. This repo can still be used to discuss community tooling.

---

Our motivations:

* Our community is too small to be decentralized, we need few places;
* Community chats are bad for discovering topics or keeping long-time living ones;
* GitHub Issues require having a GitHub account and can be scary for non-tech people;

**Comment by moul** [comment]
done, the forum is available at https://berty.community
